src/test/java/org/datadog/jmxfetch/util/MisbehavingJMXServer.java

```diff
@@ -0,0 +1,86 @@
+package org.datadog.jmxfetch.util;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import lombok.extern.slf4j.Slf4j;
+import org.datadog.jmxfetch.JMXServerControlClient;
+import org.datadog.jmxfetch.JMXServerSupervisorClient;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.lifecycle.Startable;
+
+@Slf4j
+public class MisbehavingJMXServer implements Startable {
+
+    private static final String DEFAULT_JDK_IMAGE = "base";
```

Comment:
Maybe I just haven't had my morning coffee yet, but it looks like both this constant and `this.jdkImage` are never actually read anywhere?

Reply:
Not yet, that is coming next!